architecture-decision-records/0037-disable-inactive-keycloak-users.md

```diff
@@ -0,0 +1,37 @@
+# 37. Disable inactive keycloak users
+
+**Date**: 2025-02-17
+
+## Context
+We need a reliable mechanism to identify and disable users who have not logged in for x about of days. Two main approaches were considered:
```

Comment:
Rephrase slightly: `TDR needs a ... not been active within the system for a specified period. Two main ...`